drs/drs-assembly/subscribe-model.conf

```diff
@@ -10,246 +10,86 @@ Items with the FITS= tag in the description refer to general meta-data from
 IRIS, TCS, ESW, NFIRASO, AOESW and NSCU to be associated with science data 
 (or put into the FITS header) for user post-processing.
   """
-  telemetry = [
```

Comment:
I think it would be reasonable to ask if TCS could publish an event for each of the OIWFS, ODGW, and NGS WFS. It is already performing all of the coordinate transformations for us, and will have things like ephemerides at its disposal, and so could also do things like give us the proper motions of guide stars projected into the IRIS rotated coordinate system.